test/multiverse/suites/active_record/Envfile

```diff
@@ -48,3 +48,5 @@ def gem_list(activerecord_version = nil)
     #{mysql_gem(activerecord_version)}
   RB
 end
+
+create_gemfiles(ACTIVERECORD_VERSIONS, gem_list)
```

Comment:
The `create_gemfiles` call is currently missing from dev, but will be addressed in PR https://github.com/newrelic/newrelic-ruby-agent/pull/1268. Needed this update for testing, so it is included here.

`RAILS_VERSIONS` -> `ACTIVERECORD_VERSIONS` for more appropriate naming.